src/shared/scrapers/TW/index.js

```diff
@@ -0,0 +1,114 @@
+import assert from 'assert';
+import * as fetch from '../../lib/fetch/index.js';
+import * as parse from '../../lib/parse.js';
+import datetime from '../../lib/datetime/index.js';
+
+const scraper = {
+  country: 'iso1:TW',
+  level: 'country',
+  aggregate: 'country',
+  url: 'https://covid19dashboard.cdc.gov.tw/dash3',
+  timeseries: false,
+  sources: [
+    {
+      name: 'Taiwan CDC',
+      url: 'https://sites.google.com/cdc.gov.tw/2019-ncov/taiwan',
+      description: ''
+    }
+  ],
+  certValidation: false,
+  type: 'json',
+  scraper: {
+    '0': async function() {
+      const date = process.env.SCRAPE_DATE || datetime.getYYYYMMDD();
+
+      const country = {
+        date,
+        country: this.country,
+        cases: 0,
+        recovered: 0,
+        deaths: 0,
+        tested: 0,
+        active: 0,
+        negative: 0
+      };
+
+      const schemaKeysByHeading = {
+        確診: 'cases',
+        解除隔離: 'recovered',
+        死亡: 'deaths',
+        送驗: 'tested',
+        '排除(新)': 'negative' // Test Result
+      };
+
+      const dash3options = {
+        headers: {
+          Origin: 'https://919644827-atari-embeds.googleusercontent.com'
+        }
+      };
+      const dash3 = await fetch.json(this, 'https://covid19dashboard.cdc.gov.tw/dash3', 'default', date, dash3options);
+
+      // console.info(dash3[0]);
+
+      Object.keys(schemaKeysByHeading).forEach(function(k) {
+        country[schemaKeysByHeading[k]] = parse.number(dash3[0][k]) || 0;
+      });
+
+      const testSourceSchemaKeys = {
+        通報日: 'date',
+        法定傳染病通報: 'medicalRecords', // Infections Disease Notification System http://at.cdc.tw/M4MS01
+        居家檢疫送驗: 'quarantine',
+        擴大監測送驗: 'testingCriteria', //  http://at.cdc.tw/zU3557
+        Total: 'tested'
+      };
+
+      const dash4 = await fetch.json(this, 'https://covid19dashboard.cdc.gov.tw/dash4', 'default', date);
+      const testSources = {};
+
+      Object.keys(dash4).forEach(function(i) {
+        testSources[i] = {};
+        Object.keys(dash4[i]).forEach(function(k) {
+          if (testSourceSchemaKeys[k] === 'date') {
+            testSources[i][testSourceSchemaKeys[k]] = datetime.getYYYYMMDD(dash4[i][k]);
+          } else {
+            testSources[i][testSourceSchemaKeys[k]] = parse.number(dash4[i][k]) || 0;
+          }
+        });
+      });
+
+      // console.info(testSources);
+
+      const caseSourceSchemaKeys = {
+        發病日: 'date',
+        本土感染: 'local',
+        境外移入: 'imported',
+        敦睦艦隊: 'panshi' //  http://at.cdc.tw/470o1i
+      };
+
+      const dash5 = await fetch.json(this, 'https://covid19dashboard.cdc.gov.tw/dash5', 'default', date);
```

Comment:
You sure dash 5 is `cases`? Felt like `recovered` to me. (just based on the numbers, I have no idea about the language!)

Reply:
I went over all of the strings again and it's still not the clearest it could be. 
It charts onset date of each case and where it was contracted. locally, "imported" or as part of teh goodwill fleet outbreak.